src/components/Navbar/UserOptions.js

```diff
@@ -11,9 +12,13 @@ import styles from './UserOptions.module.scss';
 const AVATAR_PLACEHOLDER = 'https://i.pravatar.cc/150?img=41';
 
 const RightItem = () => {
+  const { email = '' } = useUser();
```

Comment:
We should cover the case when `user` is `null` (e.g. in the 'forgot-password' screen the user isn't signed-in yet and thus it throws an error).

What do you think about the following change:
```javascript
const user = useUser();
...
const emailHash = md5(user?.email || '');
```

Reply:
Good catch ! @donatoaguirre24

Reply:
Actually, maybe we shouldn't render UserOptions component if the user is not signed-in.
What do you think about something like this ?
```js 
const Navbar = () =>{
const user = useUser();
return  (
  <nav className={styles.nav}>
    <div className={styles.navContent}>
      <Logo />
     { user &&  <UserOptions />}
    </div>
  </nav>
)};
```

Reply:
We want to keep it as it is for now. Mainly because UserOptions show the links to sign-in and sign-up form from the forgot-password screen. I think we added it as an example in case you want to show the navbar with some items also for unauthenticated users.

Reply:
Cool, I just added those changes and squashed both commits